index.html

```diff
@@ -8,7 +8,96 @@
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    <h1>Hello world!</h1>
+    <div class="container">
+      <div class="main">
+        <h1 id="title">Survey Form</h1>
+        <p id="description">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Maiores,
+          natus qui voluptatibus recusandae similique sunt molestias a
+          architecto necessitatibus. Nemo porro dolores magni at fugiat dolore
+          doloremque ducimus eveniet inventore!
+        </p>
+      </div>
+      <form action="" id="survey-form">
```

Comment:
- [ ] Please include a URL for the **action** attribute of the form. As a place holder, you can use the hashtag symbol **#**